reference/json/BigCommerce_Payments_API.oas2.json

```diff
@@ -1375,7 +1375,50 @@
           }
         }
       }
+    },
+    "StoredPaypalInstrument": {
```

Comment:
I don't see `StoredPaypalInstrument` in the schema that you linked. I think this should be changed to `StoredPayPalAccountInstrument`